ouroboros-network/src/Ouroboros/Network/ChainFragment.hs

```diff
@@ -702,6 +706,15 @@ joinChainFragments c1@(ChainFragment t1) c2@(ChainFragment t2) =
                                -> Just (ChainFragment (t1 FT.>< t2))
       _                        -> Nothing
 
+-- | Convert a 'ChainFragment' to a 'Chain'.
+toChain :: HasHeader block => ChainFragment block -> Chain block
+toChain = Chain.fromNewestFirst . toNewestFirst
+
+-- | Convert a 'Chain' to a 'ChainFragment'.
+fromChain :: HasHeader block => Chain block -> ChainFragment block
+fromChain = fromNewestFirst . Chain.toNewestFirst
```

Comment:
Hopefully these will only be temporary right? If so that's fine.

We're trying to move the `Chain` module into just the network test library, not the main network library. And if they'll be needed later for tests too, they can also move to the network test lib.

Reply:
Indeed, this temporary (to avoid breaking the existing tests that currently rely on `Chain`s) and should be eventually moved to the test lib.